tools/tests.py

```diff
@@ -187,12 +189,6 @@
         "dependencies": [MBED_LIBRARIES, TEST_MBED_LIB],
         "automated": True,
         "peripherals": ["analog_loop"],
-        "mcu": ["LPC1768", "LPC2368", "LPC2460", "KL25Z", "K64F", "K66F", "K22F", "LPC4088", "LPC1549",
```

Comment:
yes, I can... 
but as there is a check if analog in and out are supported or not in the test,
and test can not be executed while "peripherals" parameter is not set,
I ddidn't see any risk to remove the MCU list